src/matrix.js

```diff
@@ -61,6 +61,8 @@ module.exports.Filter = require("./filter");
 module.exports.TimelineWindow = require("./timeline-window").TimelineWindow;
 /** The {@link module:interactive-auth} class. */
 module.exports.InteractiveAuth = require("./interactive-auth");
+/** The {@link module:indexeddb-remote-worker} class. */
+module.exports.IndexedDbStoreWorker = require("./store/indexeddb-remote-worker.js");
```

Comment:
Let's be consistent on the casing please. We have `IndexedDBStore` and `IndexedDBStoreBackend`, so `IndexedDBStoreWorker` would be most preferable.